templates/ui-frameworks/lit/web-app/ui/package.json.hbs

```diff
@@ -8,67 +8,31 @@
     "build:holo": "VITE_APP_IS_HOLO=true vite build",
     "package:holo": "{{(package_manager_command package_manager "build:holo" null)}} && rimraf dist.zip && cd dist && bestzip ../dist.zip *",
   {{/if}}
-    "lint": "eslint --ext .ts,.html . --ignore-path .gitignore && prettier \"**/*.ts\" --check --ignore-path .gitignore",
```

Comment:
What's the motivation for removing the "lint" step here? I don't think it's a problem but I'm curious...

Reply:
Yes, this was controversial. I noticed that eslint is only set up by default in the lit template. And I only removed it just to keep things simple and consistent with other templates. Perhaps I could keep it or add it to all templates with a set of similar starting rules in a `.eslintrc.js.hbs` template that can be re-used across the templates perhaps. This is something I can do in a separate PR hence why I removed it here for now. What do you think?

Reply:
Okay :+1: I'm fine with removing it, I have no attachment to it.